app/src/main/java/protect/card_locker/preferences/SettingsActivity.java

```diff
@@ -75,6 +80,24 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
             // Load the preferences from an XML resource
             addPreferencesFromResource(R.xml.preferences);
 
+            SwitchPreferenceCompat appLockSwitch = findPreference(getResources().getString(R.string.settings_key_app_lock));
+            if(appLockSwitch != null){
+                appLockSwitch.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
+                    @Override
+                    public boolean onPreferenceChange(Preference preference, Object newValue) {
+
+                        boolean isSwitchOn = (boolean) newValue;
+                        if(isSwitchOn){
+                            new LockCreationDialogBuilder(getActivity())
+                                    .onCanceled(() -> { })
+                                    .onLockCreated(() -> { Toast.makeText(getActivity(), "Lock Created", Toast.LENGTH_SHORT).show(); })
```

Comment:
Don't use a hardcoded string here, use a R.string. value so it can be translated.